spec/components/avo/panel_header_component_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe Avo::PanelHeaderComponent, type: :component do
+  # it "renders something useful" do
```

Comment:
should we actually implement this spec?

Reply:
I think we should. Not sure why it's commented out.

Reply:
All of spec are made like this.

Reply:
![image](https://github.com/user-attachments/assets/beb8fd2d-5008-4bc6-9bc0-ffe9733a257d)

Reply:
> should we actually implement this spec?

Not really, it was generated while generating the component

> I think we should. Not sure why it's commented out.

It was generated during the component generation, we don't implement component tests yet so it can be removed for this PR